Add AWS-specific tuning to the OpenShift profile.

During the move of OpenShift profiles to upstream TuneD:
https://github.com/redhat-performance/tuned/pull/376

The following AWS-specific tuning for NVMe devices
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html
was intentionally omitted to keep the profiles clean from provider-specific
detail until we have a mechanism for delivering cloud-provider specific
content.  This patch preserves the old functionality until then.

diff --git a/assets/tuned/daemon/profiles/openshift/tuned.conf b/assets/tuned/daemon/profiles/openshift/tuned.conf
index af2f2bc6..9b370ea2 100644
--- a/profiles/openshift/tuned.conf
+++ b/profiles/openshift/tuned.conf
@@ -28,3 +28,7 @@ vm.max_map_count=262144
 [scheduler]
 # see rhbz#1979352; exclude containers from aligning to house keeping CPUs
 cgroup_ps_blacklist=/kubepods\.slice/
+# workaround for rhbz#1921738
+runtime=0
+
+[sysfs]
+/sys/module/nvme_core/parameters/io_timeout=4294967295
+/sys/module/nvme_core/parameters/max_retries=10
